package.json

```diff
@@ -86,7 +86,8 @@
       "valid-jsdoc": 0,
       "capitalized-comments": 0,
       "new-cap": 0,
-      "unicorn/no-process-exit": 0
+      "unicorn/no-process-exit": 0,
+      "unicorn/no-unreadable-array-destructuring": 0
```

Comment:
now that 
```
[,, nonProxiedValue] = useObservable({ value: 10 })
```
is possible, we want to disable this rule... eventually this should be rolled up into the eslint-config - https://github.com/Tram-One/eslint-config-tram-one/issues/12